cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaConfigurationDiff.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.operator.resource;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.ConfigEntry;
+import org.apache.kafka.common.config.ConfigResource;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.Set;
+
+public class KafkaConfigurationDiff {
+
+    private static final Logger log = LogManager.getLogger(KafkaConfigurationDiff.class.getName());
+    private Map<ConfigResource, Config> current;
+    private ConfigMap desired;
+
+    private static final ArrayList<String> DYNAMICALLY_CHANGEABLE_ENTRIES = new ArrayList<>(Arrays.asList(
+            "test.property.name"));
```

Comment:
I guess we have the right list somewhere?

Reply:
We have a google doc with some hints.